processor/resourcedetectionprocessor/internal/system/system.go

```diff
@@ -32,6 +32,14 @@ var (
 		featuregate.WithRegisterFromVersion("v0.89.0"),
 		featuregate.WithRegisterReferenceURL("https://github.com/open-telemetry/semantic-conventions/issues/495"),
 	)
+	hostCPUSteppingAsStringID          = "processor.resourcedetection.hostCPUSteppingAsString"
+	hostCPUSteppingAsStringFeatureGate = featuregate.GlobalRegistry().MustRegister(
+		hostCPUSteppingAsStringID,
+		featuregate.StageBeta,
```

Comment:
Let's start with alpha
```suggestion
		featuregate.StageAlpha,
```